src/app/events/event-components/event-participants-filters/event-participants-filters.component.html

```diff
@@ -0,0 +1,108 @@
+<div class="filters">
+  <h2 class="filters-header">{{ 'strings.headingFilters' | translate }}</h2>
+  <form class="filters-form" [formGroup]="filtersForm">
+    <mat-form-field id="freeText" appearance="outline" floatLabel="always" class="search-field">
+      <mat-icon matPrefix class="search-icon material-icons-round">search</mat-icon>
+      <input
+        value=""
+        type="text"
+        matInput
+        appDebounce
+        placeholder="{{ 'strings.promptEventGroupSearchField' | translate }}"
+        formControlName="freeText"
+        (debounce)="onFormChange()"
+      />
+    </mat-form-field>
+
+    <app-collapsable-box boxTitle="{{ 'headingBirthdate' | translate }}">
+      <mat-form-field appearance="outline" class="collapsable-field">
+        <mat-select
+          disableOptionCentering
+          (selectionChange)="onFormChange()"
+          formControlName="birthdateYYYY"
+          placeholder="{{ 'captions.Person.birthdateYYYY' | translate }}"
+        >
+          <mat-option value="1900">1900</mat-option>
+        </mat-select>
+      </mat-form-field>
+      <mat-form-field appearance="outline" class="collapsable-field">
+        <mat-select
+          disableOptionCentering
+          (selectionChange)="onFormChange()"
+          formControlName="birthdateMM"
+          placeholder="{{ 'captions.Person.birthdateMM' | translate }}"
+        >
+          <mat-option value="1">January</mat-option>
+        </mat-select>
+      </mat-form-field>
+      <mat-form-field appearance="outline" class="collapsable-field">
+        <mat-select
+          disableOptionCentering
+          (selectionChange)="onFormChange()"
+          formControlName="birthdateDD"
+          placeholder="{{ 'captions.Person.birthdateDD' | translate }}"
+        >
+          <mat-option value="1">1</mat-option>
+        </mat-select>
+      </mat-form-field>
+    </app-collapsable-box>
+
+    <app-collapsable-box boxTitle="{{ 'captions.Sample.pathogenTestResult' | translate }}">
+      <mat-radio-group
+        name="laboratoryResults"
+        formControlName="laboratoryResults"
+        (change)="onFormChange()"
+      >
+        <mat-radio-button value="INDETERMINATE" color="primary">
```

Comment:
You might want to use the PathogenTestResultType.INDETERMINATE for the value from the models folder.